### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by arkoort**
Fix lingvoforum.net incompatibilities

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4571,6 +4571,29 @@ CSS
...
+span.topslice
+span.botslice
+
+CSS
+body {
+  background-image: none;
+}
+
+div#header {
+    color: black;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4588]: Change it to 
```
color: ${white} !important;
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -4571,6 +4571,29 @@ CSS
...
+}
+
+div#header a:link, div#header a:visited {
+    color: black;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4592]: Change it to 
```
div#header a:link,
div#header a:visited {
    color: ${white} !important;
}
```

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -4571,6 +4571,29 @@ CSS
...
+
+INVERT
+div#header
+div#header img.avatar
+span.topslice
+span.botslice
+
+CSS
+body {
+  background-image: none;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4584]: Add `!important;` and use 4 spaces.

**Review by Gusted — CHANGES_REQUESTED** [review]
Just some code-style issues.

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (arkoort) [reply src/config/dynamic-theme-fixes.config]: done

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (arkoort) [reply src/config/dynamic-theme-fixes.config]: !important added; unfortunately neither ${white} nor ${blank} work correct, one of them gives unreadable text in dark scheme, other in light scheme. Only ugly "black" works in both schemes.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (arkoort) [reply src/config/dynamic-theme-fixes.config]: The same -- fixed all but ${white}

**Review by Gusted — CHANGES_REQUESTED** [review]
Forgot that we are migrating to no empty lines within the CSS section.
Remove L4586 and L4590

**Comment by Gusted** [comment]
Thanks!

**Pull request merged**